uptane-standard.md

```diff
@@ -572,7 +583,7 @@ A Director repository MUST conform to the following six-step process for directi
     * The signature of each secondary's contribution to the manifest matches the ECU key of that secondary.
 1. The Director extracts information about currently installed images from the vehicle version manifest. Using this information, it determines if the vehicle is already up-to-date, and if not, determines a set of images that should be installed. The exact process by which this determination takes place is out of scope of this standard. However, it MUST take into account *dependencies* and *conflicts* between images, and SHOULD consult well-established techniques for dependency resolution.
 1. The Director MAY encrypt images for ECUs that require it.
-1. The Director generates new metadata representing the desired set of images to be installed in the vehicle, based on the dependency resolution in step 4. This includes targets ({{targets_meta}}), snapshot ({{snapshot_meta}}), and timestamp ({{timestamp_meta}}) metadata. It then sends this metadata to the primary as described in {{download_meta_primary}}.
+1. The Director generates new metadata representing the desired set of images to be installed on the vehicle, based on the dependency resolution in step 4. This includes targets ({{targets_meta}}), snapshot ({{snapshot_meta}}), and timestamp ({{timestamp_meta}}) metadata. It then sends this metadata to the primary as described in {{download_meta_primary}}.
```

Comment:
"on the vehicle" sounds a bit funny to me here as well, but I'm less sure about this one.

Reply:
I'm conceding all the "ins" except this one. In this case, we are talking about placing the images on the car, and not "in" the repo. The car is hardware. Hence, I think this one stay as "on" at least for now.